src/MsgType.jl

```diff
@@ -0,0 +1,27 @@
+
+const MsgType = (
+    from_host_call_with_response = UInt8(1),
+    from_host_call_without_response = UInt8(2),
+    from_host_interrupt = UInt8(20),
+    ####
+    from_worker_call_result = UInt8(80),
+    from_worker_call_failure = UInt8(81),
+    ###
+    special_serialization_failure = UInt8(100),
+)
+
+const MsgID = UInt64
+
+const BUFFER_SIZE = 4 * Base.SZ_UNBUFFERED_IO
```

Comment:
@Pangoraw Did `4 *` work better in your tests?

Reply:
Testing with the old `BufferedIO`, I found `4 * 65536 = 1048576` to also work at the same performance level as `65536`, but `8 * 65536` would start affecting performance:

[`a631a0a` (#32)](https://github.com/JuliaPluto/Malt.jl/pull/32/commits/a631a0a655bd37dec11776c0fd02df8942acb7fd#diff-3ec51f087381c93e76dd3413ac333aa76c9ad3b601fcf2cd2c59bf6cc2e568c0L111-L128)

Reply:
Maybe we can set it to 65536 then?